python/py-behave/Portfile

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           python 1.0
+
+github.setup        behave behave 1.2.6 v
+revision            0
+name                py-${github.project}
+categories-append   devel
+
+license             BSD
+maintainers         nomaintainer
+platforms           darwin
+
+description         behave is behavior-driven development, Python style.
+
+long_description    Behavior-driven development (or BDD) is an agile\
+                    software development technique that encourages\
+                    collaboration between developers, QA and\
+                    non-technical or business participants in a\
+                    software project. behave uses tests written in a\
+                    natural language style, backed up by\
+                    Python code.
+
+checksums           rmd160  a78eda96ca951924b19f20c4589f82f4d2d91de0 \
+                    sha256  ea4351f89631c9a0f439d831c3d27c32f9f33d8965e1ec47bf4cac8800add7bd \
+                    size    701895
+
+supported_archs     noarch
+
+python.versions      27 35 36 37 38
+
+if {${name} ne ${subport}} {
+    depends_build-append \
```

Comment:
this should be ```depends_lib``` since it's also used at runtime due to the ```console_scripts```

Reply:
done